tsconfig-type-test.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "target": "es2018",
+    "module": "esnext",
+    "isolatedModules": true,
+    "strict": true,
+    "verbatimModuleSyntax": true,
+    "noEmit": true,
+    "allowSyntheticDefaultImports": true,
+    "moduleResolution": "bundler",
+    "noImplicitAny": false,
```

Comment:
We should use `noImplicitAny: true` which should be default when `strict: true`

Reply:
This would require a lot of additional type casts `@ts-expect-error` in places where the current API is nearly impossible to statically type.